addons/planner_project/static/src/js/planner_project.js

```diff
@@ -0,0 +1,63 @@
+(function(){
+    "use strict";
+    var _t = openerp.web._t;
+
+    openerp.planner.PlannerDialog.include({
+        prepare_planner_event: function() {
+            var self = this;
+            this._super.apply(this, arguments);
+            if(self.planner['planner_application'] == 'planner_project') {
+                var stages = {
+                    'development_process': [
```

Comment:
@jem-odoo : Current code for this works fine. The given solution will enlarge the code. Should I keep it as it or make according to {'input_element_stage_1' : _t('validation'), 'input_element_stage_2' : _t('Development'), .... }